frontend/src/scalars/Scalars.vue

```diff
@@ -39,14 +32,15 @@ import {getPluginScalarsTags, getRuns} from '../service';
 import {debounce, flatten, uniq, isArray} from 'lodash';
 import autoAdjustHeight from '../common/util/autoAdjustHeight';
 
-import Config from './ui/config'
+import Config from './ui/Config.vue'
```

Comment:
You can also do `import ui-config from './ui/Config.vue'`
and simply do components: {
  ui-config
}

Reply:
hmm...compiler complains when I do import ui-config, "-" is not allowed in this case